src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,128 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.tasklist.Tasklist;
+
+import java.lang.reflect.Array;
```

Comment:
Are you using this import, or just the ArrayList import? 

I just did a quick skim through so I might have missed it. So if you have used it, just ignore my comment. It is good to make sure not include unused imports. :)